.github/workflows/release.yml

```diff
@@ -11,24 +11,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: '3.x'
-      - name: Install poetry
-        run: |
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
-          echo "PATH=$HOME/.poetry/bin:$PATH" >> $GITHUB_ENV
-      - name: Build distributions
-        run: |
-          poetry build
-      - name: Upload distribution artifacts
-        uses: actions/upload-artifact@v2
+      - name: "Build package using Poetry and store result."
```

Comment:
@jjmerchante I know this review comment might be silly :speak_no_evil: 
Just to maintain the uniformity, do you mind removing the quotes and full stop. I mean we haven't added it anywhere in the workflows till now. The name key takes the value as a string, so we need not specify it.

I'm sorry for being so picky :sweat_smile::pray:

Reply:
Yes, I didn't notice it! :+1:

Reply:
Thanks JJ!! :partying_face: